### ISSUE TITLE
How to uninstall/purge hasura?

### ISSUE THREAD

**Issue by jflambert** [hasura/graphql-engine#3923]
Dear Hasura people, we're using your amazing hasura.cli-migrations docker container but as part of a pipeline, I would need to completely kill any hasura objects in my database in order to rebuild them. Having some dependency issues at the moment (specifically INSTEAD OF triggers that seem to get created when granting some partial INSERT permissions)

Is it as simple as dropping the `hdb_views` and `hdb_catalog` schemas?

I would suggest adding this information to your also-amazing documentation.

Thanks!

**Comment by tirumaraiselvan** [comment]
@jflambert That is right. We should definitely add how to uninstall hasura in the docs.

cc: @marionschleifer

**Comment by 0x777** [comment]
@jflambert 

The following should do the trick:
```sql
drop schema hdb_views cascade;
drop schema hdb_catalog cascade;
```

**Comment by Smartmind12** [comment]
@marionschleifer & Maintainers, Hey, I have created a PR to solve this issue as a part of Hacktoberfest 2022 for which i forgot to get it assigned by the Maintainers...pardon me. Kindly review the same and recommend if any additions are required.
